installer/settings/common.py

```diff
@@ -161,7 +161,7 @@
 JOB_DETAIL_TYPE = "Paladin Cloud Job Scheduling Event" #please do not change these, as this may stop scheduling of jobs
 JOB_SOURCE = "paladincloud.jobs-scheduler"
 
-CURRENT_RELEASE = "v3.0.1"
+CURRENT_RELEASE = "v3.0.3"
 DB_USERNAME = "paladin" 
 DB_PASSWORD = "***PALADIN***" #Only printable ASCII characters besides '/', '@', '"', ' ' may be used.
```

Comment:
Remove this from installer. And make it configurable.

Reply:
It is already a configuration value in local.py

if user is not willing to update rds username and rds pwd it will pick form here